grpc-interface-support/src/main.py

```diff
@@ -54,34 +143,76 @@ def create_service_sdk_dir(proto_file_handle: proto.ProtoFileHandle) -> str:
     return service_sdk_path
 
 
-def generate_single_service(
+def get_proto_include_dir(path: str) -> str:
+    if os.path.isdir(path):
+        return path
+    elif os.path.isdir(os.path.join(get_workspace_dir(), path)):
+        return os.path.join(get_workspace_dir(), path)
+    elif os.path.isdir(os.path.join(get_project_cache_dir(), "downloads", path)):
+        return os.path.join(get_project_cache_dir(), "downloads", path)
+    else:
+        raise FileNotFoundError(f"Directory {path} not found!")
+
+
+def generate_services(
     factory: GrpcServiceSdkGeneratorFactory, if_config: Dict[str, Any]
 ) -> None:
-    """Generate an SDK for a single service.
+    """Generate an SDK for service(s).
 
     Args:
         factory (GrpcPackageGeneratorFactory):
             The factory from which to generate an SDK generator for a single service.
         if_config (Dict[str, Any]): The grpc-interface config.
     """
-    print(
-        f"Generating service SDK for {create_truncated_string(if_config['src'], 100)!r}"
-    )
-    proto_file_handle = proto.ProtoFileHandle(obtain_local_file_path(if_config["src"]))
-    service_sdk_dir = create_service_sdk_dir(proto_file_handle)
 
+    root = if_config.get("rootPath", None)
```

Comment:
root is a bit misleading IMO, what about something like 
```suggestion
    path_in_zip = if_config.get("rootPath", None)
```